make/jdk/src/classes/build/tools/generatezip/GenerateZip.java

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2021, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello @andrew-m-leonard, for a new file, the year should just be stated once. So just `2021,`